model/gen-ai/spans.yaml

```diff
@@ -147,9 +147,10 @@ groups:
         requirement_level:
           conditionally_required: If not default (443).
 
-  - id: span.gen_ai.client.create_agent
+  - id: span.gen_ai.create_agent.client
```

Comment:
Was this change intended?

Reply:
yep, the id format now required `span.*.{kind}` if kind is specified